README.md

```diff
@@ -1,76 +1,89 @@
-* If you are testing the logger components, you must enable `DaemonSet` experimental APIs via `--runtime-config=extensions/v1beta1/daemonsets=true`
+* Create a new application
+* Delete an application
+* Scale an application
+* Configure an application
+* Create a new user
 
-Next, install [helm](http://helm.sh). Next, add the deis repository to your chart list:
+# Development
 
-```console
-$ helm repo add deis https://github.com/deis/charts
-```
+The Deis project welcomes contributions from all developers. The high level process for development matches many other open source projects. See below for an outline.
 
-Then, install Deis!
+* Fork this repository
+* Make your changes
+* [Submit a pull request][prs] (PR) to this repository with your changes, and unit tests whenever possible.
+  * If your PR fixes any [issues][issues], make sure you write Fixes #1234 in your PR description (where #1234 is the number of the issue you're closing)
+* The Deis core contributors will review your code. After each of them sign off on your code, they'll label your PR with LGTM1 and LGTM2 (respectively). Once that happens, you may merge
 
-To work off the latest stable
+## Prerequisities
 
-```console
-$ helm install deis/deis
-```
+### Kubernetes
 
-To work off the latest development version
+In order to do development on this component, you'll need a working Kubernetes cluster. If you don't have one follow the [installation instructions][install-k8s] and note that Workflow currently targets version 1.1 with the following requirements:
 
-```console
-$ helm install deis/deis-dev
-```
+* Docker's `insecure-registry` parameter must include the subnets used by your Kubernetes installation
+* If you are testing the logger components, you must enable `DaemonSet` experimental APIs via `--runtime-config=extensions/v1beta1/daemonsets=true`
 
-Complete instructions for installing and managing a Deis cluster are
-available at https://github.com/deis/docs-v2
+### Helm
 
-If you want to retrieve the latest client dev build for OS X or Linux, download the client:
+After you have a working Kubernetes cluster, install [helm](http://helm.sh) and run the following commands to add the Deis chart repository and install Deis to your new cluster:
 
 ```console
-$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash
+helm repo add deis https://github.com/deis/charts
+helm install deis/deis-dev
 ```
 
-If you want to hack on a new feature, build the deis/workflow image and push it to a Docker
-registry. The `$DEIS_REGISTRY` environment variable must point to a registry accessible to your
-Kubernetes cluster. You may need to configure the Docker engines on your Kubernetes nodes to allow
-`--insecure-registry 192.168.0.0/16` (or the appropriate address range).
+Note that to work off the latest stable release, change the `helm install deis/deis-dev` command to `helm install deis/deis`.
```

Comment:
@mboersma @sgoings I'll omit for now, and we can modify `README`s across the board once we have a latest stable beta.